pynamodb/indexes.py

```diff
@@ -147,17 +150,18 @@ def _get_schema(cls) -> Dict:
             'projection': {
                 PROJECTION_TYPE: cls.Meta.projection.projection_type,
             },
+            'attribute_definitions': {},
         }
+
         for attr_cls in cls.Meta.attributes.values():
-            if attr_cls.is_hash_key:
-                schema['key_schema'].append({
+            if attr_cls.is_hash_key or attr_cls.is_range_key:
+                schema['attribute_definitions'][attr_cls.attr_name, attr_cls.attr_type] = {
```

Comment:
Now indexes will return "attribute definitions" even though they only roll up into the table's.